### PR TITLE
Update Using Notify content

### PR DESCRIPTION
**Pull request by thomchambers**
Editing and updating content based on user support tickets and new features. Part of the general content batch over the last couple of days.

@quis the 'account' thing pops up here again, as in Pricing. Again, happy to discuss/amend.

### CODE REVIEW

**Comment by quis** [comment]
@govuk-notify-jenkins test this please

File: app/templates/views/using-notify.html [hunk 1]
```diff
@@ -13,26 +13,18 @@
   </div>
   <div class="column-two-thirds">
 
-    <h1 class="heading-large">Using Notify</h1>
```
Comment (quis) [inline app/templates/views/using-notify.html]: This page still needs a H1

Comment (thomchambers) [reply app/templates/views/using-notify.html]: Whoops

**Comment by quis** [comment]
@govuk-notify-jenkins test this please

**Pull request merged**